### PR TITLE
Live Schedule: Add tests for "current session" functionality

### PR DESCRIPTION
**Pull request by ryelle**
This will test the `getCurrentSessions` function to make sure it returns the expected sessions at given times. This will be helpful once we start changing the Live Schedule functionality (adding session durations, tweaking the date conversion #226, other fixes like #260, #261, #279).

### How to test the changes in this Pull Request:

1. Update packages by running `yarn` (new local packages are needed to run the tests)
2. Run the tests with `yarn workspace wordcamp-blocks run test`

### CODE REVIEW

File: public_html/wp-content/mu-plugins/blocks/source/blocks/live-schedule/data/tests/__mock__/session_track.json [hunk 1]
```diff
@@ -1 +1,38 @@
...
\ No newline at end of file
+[
+  {
```
Comment (iandunn) [inline public_html/wp-content/mu-plugins/blocks/source/blocks/live-schedule/data/tests/__mock__/session_track.json:2]: Great idea to make this more readable 👍

**Review by iandunn — APPROVED** [review]
Code LGTM, and the tests pass locally 👍 

---

_Non-blocking aside:_ Should `yarn test` also work? 

Thats [in our docs](https://github.com/WordPress/wordcamp.org/blob/e6f354e/public_html/wp-content/mu-plugins/blocks/readme.md), but doesn't WFM:

```sh
> yarn test
yarn run v1.21.1
error Command "test" not found.
```

**Comment by ryelle** [comment]
> Should yarn test also work?

@iandunn What directory are you in when you run that? If you're in `mu-plugins/blocks` that should work. If you're not, you need to tell yarn which workspace to run in, because we don't have a global `test` command.

**Comment by iandunn** [comment]
Ah, that was probably it. Thanks!

**Pull request merged**